db/migrate/20190304151614_remove_default_satisfaction.rb

```diff
@@ -0,0 +1,17 @@
+class RemoveDefaultSatisfaction < ActiveRecord::Migration[5.2]
+  def up
+    change_column :facts_metrics, :satisfaction, :float, null: true
+    change_column_default :facts_metrics, :satisfaction, nil
+
+    change_column :aggregations_monthly_metrics, :satisfaction, :float, null: true
+    change_column_default :aggregations_monthly_metrics, :satisfaction, nil
+  end
+
+  def down
+    change_column :facts_metrics, :satisfaction, :float, null: false
+    change_column_default :facts_metrics, :satisfaction, 0.0
+
+    change_column :aggregations_monthly_metrics, :satisfaction, :float, null: false
+    change_column_default :aggregations_monthly_metrics, :satisfaction, 0.0
+  end
```

Comment:
I think Rails see this as an irreversible migration. In which case you could simplify the migration to something like:
```ruby
  def change
    change_column_default :aggregations_monthly_metrics, :satisfaction, nil
    change_column_null :aggregations_monthly_metrics, :satisfaction, true

    change_column_default :facts_metrics, :satisfaction, nil
    change_column_null :facts_metrics, :satisfaction, true
  end
```

or maybe worth leaving a comment?

Reply:
I hit the irreversible problem when it was only within a `change`, which is why explicitly setting what the defaults will be in `up` and `down` will work (and I tested locally).